libioc/Pkg.py

```diff
@@ -67,6 +67,7 @@ def fetch(
         event_scope: typing.Optional['libioc.events.Scope']=None
     ) -> typing.Generator[libioc.events.IocEvent, None, None]:
         """Fetch a bunch of packages to the local mirror."""
+        return
```

Comment:
is this a good idea?

Reply:
No, it was not! We used this to reduce the delay while debugging.